postgresql-examples/atomic-batch-operation/README.md

```diff
@@ -0,0 +1,42 @@
+# Overview
+
+The `atomic-batch-operation` project demonstrates how to use the PostgreSQL client to execute a batch of DDL/DML operations with the help of a `transaction` to achieve the atomic behaviour.
+
+# Prerequisite
+
+* Install the PostgreSQL server and create a database 
+
+* Add required configurations in the `config.toml` file 
+
+* Download the PostgreSQL database driver JAR and update the path and version of the jar in the `Ballerina.toml` file
```

Comment:
If we download the driver JAR, mentioning only the path is enough. or we can add maven dependency config with version